modules/farm/farm_crop/farm_crop.farm_quick.planting.inc

```diff
@@ -237,13 +237,17 @@ function farm_crop_planting_quick_form($form, &$form_state) {
     implode(', ', $crop_tags),
   );
 
+  // Cannot change the default value without unsetting form_state input
```

Comment:
This solution was found [here](https://drupal.stackexchange.com/questions/251056/set-default-form-values-after-ajax-callback)

Reply:
Good find!  And yea I've had to use the same "solution" myself in the soil amendment and soil disturbance quick forms. Drupal 7 #ajax has some quirks to it, for sure.